pj-dao/src/contract.rs

```diff
@@ -627,18 +625,16 @@ fn mint_dice_nft_handle_msg(
     mint_to: &HumanAddr,
     block_time: u64,
     private_metadata: Option<Metadata>,
-) -> (NftHandleMsg, String) {
+) -> NftHandleMsg {
     let viewing_key =
```

Comment:
we need to remove this. Please update to use  
```
   NftHandleMsg::MintDiceNft {
       
        owner: mint_to.to_owned(),
   
        private_metadata,

    };
    }
    ```
    
    I will align on NFT end.

Reply:
https://github.com/iorveth/scrt-poker-joker/pull/36/commits/6e664d65bed985d99431f7c4622c0aab705eb18b